src/source_map_utils.ts

```diff
@@ -30,7 +30,8 @@ export function extractInlineSourceMap(source: string): string {
     previousResult = result;
     result = inlineSourceMapRegex.exec(source);
   } while (result !== null);
-  const base64EncodedMap = previousResult![1];
+  if (!previousResult) return null;
```

Comment:
> I agree that it's bad to have an api that forces you to call a guard function before you call another function, but I don't think that what's happening here.
> My goal with extractInlineSourceMap is that you only call it when you're sure that the file you're calling it on has an inline source map and that it throws nasty errors if you didn't figure that out correctly.

This sounds contradictory to me. How can you be sure that you can call this function? By calling the guard function.

I think it's fine to have an "attempt to extract a source map, and if you don't find one, don't". That's what the code would do if it didn't fail, wouldn't it? Is the actual problem just that the two regexs don't match?